andrey/NewHW1/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<body> 
+
+<head> 
+    <script src="NewTask1.js"></script>
+
+</head>

```

Comment:
head element is not allowed inside of the body element. Please investigate any html document to find a right place of head tag